.licenserc.yaml

```diff
@@ -1,4 +1,14 @@
 header:
+
+  license:
+    copyright-owner: Canonical Ltd.
+    content: |
+      Copyright (C) 2014 [owner]
+      See LICENSE file for licensing details.
```

Comment:
This license file is no longer included in out repository, you'll probably want to include the whole harcoded header here. Also, is this a GPT3 license? https://www.apache.org/licenses/GPL-compatibility.html

Reply:
It is a GPLv3 License as stated in the link. How should we proceed in this case?